eng/Workarounds.targets

```diff
@@ -54,21 +49,6 @@
     </KnownFrameworkReference>
   </ItemGroup>
 
-  <!-- Workaround until we can build our repo with a 3.0 Preview 7 SDK which has https://github.com/dotnet/core-sdk/pull/2401. -->
-  <ItemGroup Condition="'$(PreReleasePreviewNumber)' == '7' ">
-    <KnownFrameworkReference Update="Microsoft.NETCore.App">
-      <!--
-        Temporary until we can update to build with the Preview7 SDK .
-        Using PreReleasePreviewNumber to time-bomb this so we can remove this before RTM.
-
-        This workaround was also applied to src\ProjectTemplates\test\Infrastructure\TemplateTests.props.in.
-      -->
-      <RuntimePackNamePatterns>Microsoft.NETCore.App.Runtime.**RID**</RuntimePackNamePatterns>
-      <AppHostPackNamePattern />
-      <AppHostRuntimeIdentifiers />
-    </KnownFrameworkReference>
-  </ItemGroup>
-
   <!-- Work around https://github.com/dotnet/cli/issues/11378. -->
```

Comment:
Remove this bit as well. This old workaround is not necessary now that we're using the preview 7 SDK. The last few comments about a new problem in the preview 8 SDK are worked around [above](https://github.com/aspnet/AspNetCore/pull/12812/files#diff-443ee38227003305d126960ccf383455R45-R50)